### PR TITLE
Add Missing Guild Implementations

### PR DESCRIPTION
**Pull request by Helloyunho**
This pr adds missing guild implementations (which are written in #86).

If you want to check what things are implemented right now, check #86.

### CODE REVIEW

File: src/managers/roles.ts [hunk 1]
```diff
@@ -80,4 +86,17 @@ export class RolesManager extends BaseManager<RolePayload, Role> {
...
+    if (role instanceof Role) {
+      role = role.id
+    }
+    const resp = await this.client.rest.patch(
+      GUILD_ROLE(this.guild.id, role),
+      options
+    )
+
+    const result = new Role(this.client, resp, this.guild)
+    return result
```
Comment (AkiaCode) [inline src/managers/roles.ts:100]:
```javascript
return new Role(this.client, resp, this.guild)
```

**Review by AkiaCode — APPROVED** [review]

File: src/managers/roles.ts (hunk 1 above)
Comment (Helloyunho) [reply src/managers/roles.ts]: Fixed from e99bf61f3a6372732fa8ba14be5230668a6234f0

File: src/structures/guild.ts [hunk 2]
```diff
@@ -385,6 +385,42 @@ export class Guild extends Base {
...
+  async prune(
+    days?: number,
+    computePruneCount?: undefined,
+    includeRoles?: Array<Role | string>
+  ): Promise<number>
+  async prune(
+    days?: number,
+    computePruneCount?: boolean,
+    includeRoles?: Array<Role | string>
+  ): Promise<number | null> {
```
Comment (AkiaCode) [inline src/structures/guild.ts:408]: What...

**Review by AkiaCode — CHANGES_REQUESTED** [review]

File: src/structures/guild.ts (hunk 2 above)
Comment (Helloyunho) [reply src/structures/guild.ts:408]: This is to show user different return types per `computePruneCount`. I didn't have better idea than that.

**Review by AkiaCode — APPROVED** [review]

**Comment by Helloyunho** [comment]
I made this pr out of draft since @DjDeveloperr made a pr that covers the rest of things.(#88)

**Review by DjDeveloperr — APPROVED** [review]
LGTM - ready to land!

**Comment by Helloyunho** [comment]
umm I should do tests tho but anyway

**Pull request merged**